.github/pull_request_template.md

```diff
@@ -7,28 +7,41 @@ Remove the checklist after fulfilling all the relevant points, and before creati
 
 ### Checklist
 
-__Thank you for this this PR! Please consider the following:__
+__Thank you for this PR! Please consider the following:__
 
 * To the _Community_ :heart::
-    * Please link to an issue ticket where your code change has been greenlit, otherwise it's unlikely it can be merged.
-    * Use a descriptive title: {task_name} (closes #{issue_number}), e.g.: `Use logger (closes # 41)`.
-    * If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
-    * The PR can not contain changes in localization files, e.g. `values-[LANGUAGE-CODE]/strings/*` and `/` or `assets/*` (see [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332) for further information).
+    * Please link to an issue ticket where your code change has been greenlit, otherwise it's
+      unlikely it can be merged.
+    * Use a descriptive title: {task_name} (closes #{issue_number}),
+      e.g.: `Use logger (closes # 41)`.
+    * If this PR comes from a
+      fork,please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
+    * The PR can not contain changes in localization files, e.g. `values-[LANGUAGE-CODE]/strings/*`
+      and `/`
+      or `assets/*` (see [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332) for
+      further information).
+    * Before submitting a PR, please make sure that you target the active `release/*.*.*` branch
+      which can be determined by the active milestone on GitHub
+    * Include `community` in PR title such as "PR is fixing issue ### (community)"
```

Comment:
```suggestion
    * Include `COMMUNITY` in PR title such as "PR fixes issue ### (COMMUNITY)"
```
1. In the past mostly "COMMUNITY" was used, sometimes "Community" but never "community" (in lower case).
2. "is fixing" is a problematic grammar usage. "fixes" is timeless and better.